mbed_greentea/mbed_greentea_cli.py

```diff
@@ -585,6 +588,13 @@ def main_cli(opts, args, gt_instance_uuid=None):
                                          enum_host_tests_path=enum_host_tests_path,
                                          verbose=opts.verbose_test_result_only)
 
+        # Some error in htrun, abort test execution
+        if isinstance(host_test_result, int):
+            # int(host_test_result) > 0 - Call to mbedhtrun failed
+            # int(host_test_result) < 0 - Something went wrong while executing mbedhtrun
+            return int(host_test_result)
```

Comment:
Probably fine to reduce to `return host_test_result` since it is already an `int`